tests/utils_tests/test_http.py

```diff
@@ -268,6 +268,18 @@ def test_parsing_rfc850(self):
         parsed = parse_http_date('Sunday, 06-Nov-94 08:49:37 GMT')
         self.assertEqual(datetime.utcfromtimestamp(parsed), datetime(1994, 11, 6, 8, 49, 37))
 
+    def test_parsing_rfc850_year_50(self):
+        year = datetime.now().year % 100
+        dates = (
+            ('Monday, 31-Dec-18 08:49:37 GMT', datetime(2018, 12, 31, 8, 49, 37)),
```

Comment:
At what point will this be interpreted as 2118? Far enough into the future that it doesn't matter much? Might want to include a comment about that.